contracts/src/libraries/Transcript.sol

```diff
@@ -59,13 +139,37 @@ library Transcript {
         appendGroupElement(self, comm);
     }
 
+    // @dev This function computes keccak256(bytes32 a, bytes b)
+    function computeHash(bytes32 a, bytes memory b) public pure returns (bytes32 result) {
+        assembly {
+            // Load the length of 'b'
+            let bLength := mload(b)
+
+            // Calculate total data length (32 bytes for 'a' + length of 'b')
+            let totalLength := add(32, bLength)
```

Comment:
let's inline this on Line 164 since this is only used once.
simply `result:=keccak256(data, add(32, bLength))`

Reply:
Yes, done  in 0a2c9134.